salesforce-backed-customer-insight-agent/sk_connectors.py

```
@@ -0,0 +1,184 @@
+from typing import Any, Dict, Optional
+
+from scalekit import ScalekitClient
+from scalekit.core import ScalekitException
+
+from settings import Settings
+
+
+class ScalekitConnector:
+    def __init__(self) -> None:
+        if not Settings.SCALEKIT_CLIENT_ID or not Settings.SCALEKIT_CLIENT_SECRET:
+            raise ValueError("Scalekit credentials not configured")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)